src/pages/MarketPage/components/Product/Product.tsx

```diff
@@ -4,7 +4,15 @@ import "./Product.css";
 import defaultImg from "../../../../images/defaultImg.jpeg";
 import { Link } from "react-router-dom";
 
-function Product({ product }) {
+interface ProductInfo {
+  images: string[];
+  name: string;
+  price: number;
+  favoriteCount: number;
+  id: string;
+}
+
+function Product({ product }: { product: ProductInfo }) {
   const { images, name, price, favoriteCount, id } = product;
   console.log(`${id}`);
```

Comment:
```suggestion
  console.log(`${id}`);
```
여기도 로컬에서 확인이 끝났다면 지워주시는게 좋습니다 :-)